week-4/Homework/mandatory/2-writers.js

```diff
@@ -42,3 +42,4 @@ let writers = [
 /*
 If you want an extra challenge, only `console.log()` the writers that are alive.
 */
+let writerAlive= writers.filter(n=>n.alive === true).forEach(n=> console.log(`Hi, my name is ${n.firstName} ${n.lastName}. I am ${n.age} years old, and work as a ${n.occupation}."`))

```

Comment:
Excellent use of `.filter` and `.forEach`, and well done on completing the extra challenge.